pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "memreps"
 version = "0.1.0"
 description = "Algorithm for learning from comparision and membership queries."
-authors = ["Marcell Vazquez-Chanlatte <mvc@linux.com>"]
+authors = ["anonymous"]
```

Comment:
I'm abit confused by this? The github is public and not anonymous? Is this for the artifact only?

Reply:
ya whoops was for artifact. will remove